docs(batcher): document how to configure custom aggregators #989 (#1100)
open-telemetry/opentelemetry-js: doc/batcher-api.md

# Batcher API Guide

> The batcher has two responsabilities: choosing which aggregator to choose for a metric instrument and store the last record for each metric ready to be exported.

Reviewer: ```suggestion
The batcher has two responsibilities: choosing which aggregator to choose for a metric instrument and store the last record for each metric ready to be exported.
```

Reviewer: Pls add link to Batcher class: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-metrics/src/export/Batcher.ts

## Selecting a specific aggregator for metrics

> Sometimes you may want to use a specific aggregator for one of your metric, export a average of the last X values instead of just the last one.

Reviewer: ```suggestion
Sometimes you may want to use a specific aggregator for one of your metric, export an average of the last X values instead of just the last one.
```

> Here is what a aggregator that does that would looks like:

Reviewer: ```suggestion
Here is what an aggregator that does that would look like:
```
